config/locales/en/lookup.yml

```diff
@@ -0,0 +1,14 @@
+en:
+  coronavirus_local_restrictions:
+    lookup:
+      input_label: Enter your postcode
+      hint_text: For example SW1A 2AA
+      button_text: Find
+      link_text: Find a postcode on Royal Mail's postcode finder
+      href: http://www.royalmail.com/find-a-postcode
+      title: Find out the coronavirus restrictions in a local area
+      inset_text: There are different restrictions in Scotland, Wales and Northern Ireland.
```

Comment:
As the postcode-lookup view is completely separate from the rest of the coronavirus templates, there's probably and argument for splitting these into their own locale file.

Reply:
ah I didn't put in its own file, I put it under a separate key 😅 ... is that ok?

Reply:
Yeah. Considering the time constraints it's a definite nitpick. We can separate the files when things have calmed down a bit.